src/chunkserver/trash.h

```diff
@@ -164,6 +166,8 @@ class Trash {
     // 回收站中chunk的个数
     Atomic<uint32_t> chunkNum_;
```

Comment:
Does `chunkNum_` is the number of chunk file and snapshot file ?
When `chunkFilePool_` and `walPool_` refer to the same pool, should `chunkNum_` also include wal chunk file ?

Reply:
This issue is left to fix by future PR